ios_automation/page_action/category_page.py

```diff
@@ -59,6 +67,26 @@ def click_category(wd, category_name):
     ialc(wd, category_name)
 
 
+'''
+카테고리 PLP
+'''
+
+
+def check_not_login_user_recommended_tab(wd):
+    try:
+        ial(wd, 'c_당신을 위한 추천상품')
+        print('비로그인 유저 추천 페이지 타이틀 확인')
+    except NoSuchElementException:
+        print('비로그인 유저 추천 페이지 타이틀 확인 실패')
+        raise Exception('비로그인 유저 추천 페이지 타이틀 확인 실패')
+
+
+def save_webview_category_product_name(wd):
+    product = wd.find_element(AppiumBy.CSS_SELECTOR, '[class="css-j22x2q e13ypwtz2"]')
```

Comment:
class 이름은 . 으로 이어 써주시는것이 정확한데 css seletor에도 적용되는 것인지 확실하지 않네요. 스페이스가 있을 경우 앞의 class 이름만 인식해서 찾게 되기때문에 상황에 따라서 다른 요소가 찾아질 수도 있어요. 이건 확인한번 해보시고 둘다 된다면 . 으로 이어서 써주시는것이 좋습니다

```suggestion
    product = wd.find_element(AppiumBy.CSS_SELECTOR, '[class="css-j22x2q.e13ypwtz2"]')
```